README.md

```diff
@@ -21,17 +21,15 @@ This project based on the linter configuration. setup project built with html an
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
 ### Setup
 
-### Install
-
-### Usage
+To have a local copy of this project, follow theese simple steps:
 
-### Run tests
+1. Open the terminal comand line window
+2. `git clone https://github.com/Kakalanp/Portfolio-Joaquin.git`

```

Comment:
- [ ] This link is not correct as it has someone else's name and Portfolio instead of your repo. Here is your link `https://github.com/HectorTorresE/Awesome-books.git` in case you don't know 😉.

- [ ] Let's remove these lines as they are just instructions from the README template. See them in the image below.

<img width="475" alt="Loom_qtv8JnAvC7" src="https://user-images.githubusercontent.com/81468675/153102205-c3f8cc88-6871-4940-8ee8-724d39c0bf6d.png">